tasks/bump.js

```diff
@@ -74,15 +75,26 @@ module.exports = function(grunt) {
     var VERSION_REGEXP = /([\'|\"]?version[\'|\"]?[ ]*:[ ]*[\'|\"]?)([\d||A-a|.|-]*)([\'|\"]?)/i;
 
 
-    // GET VERSION FROM GIT
-    runIf(opts.bumpVersion && versionType === 'git', function(){
-      exec('git describe ' + opts.gitDescribeOptions, function(err, stdout, stderr){
+    // GET VERSION
+    runIf(opts.bumpVersion, function(){
+      // default to git
+      var execCmd = 'git describe ' + opts.gitDescribeOptions;
```

Comment:
this should be part of the if/else opts.vcs block. if git set these variables else if hg set these else fail! then svn could potentially be added as well